spec/requests/charges_spec.rb

```diff
@@ -190,6 +191,56 @@
         end
       end
 
+      context 'with a project_id' do
```

Comment:
What happens if it has both?

Reply:
Just added the test, works as expected by returning a 422 error.

I think it should be good to go!